lib/ui/src/components/preview/tools/copy.tsx

```diff
@@ -9,12 +9,11 @@ import { stringifyQueryParams } from '../utils/stringifyQueryParams';
 const { PREVIEW_URL } = global;
 
 const copyMapper = ({ state }: Combo) => {
-  const { storyId, refId, refs } = state;
-  const ref = refs[refId];
+  const { storyId, location } = state;
+  const originUrl = location.origin;
 
   return {
-    refId,
-    baseUrl: ref ? `${ref.url}/iframe.html` : (PREVIEW_URL as string) || 'iframe.html',
+    baseUrl: originUrl ? `${originUrl}/iframe.html` : (PREVIEW_URL as string) || 'iframe.html',
```

Comment:
@ndelangen I suspect the new code doesn't handle composition ... wdyt?

Reply:
yeah, this would be a regression